swdb_2019_tools/find_soma_synapses.py

```diff
@@ -0,0 +1,62 @@
+import numpy as np
+import pandas as pd
+from analysisdatalink.datalink_ext import AnalysisDataLinkExt as AnalysisDataLink
+
+def find_soma_synapses(neuron_id, radius, synapses_name = 'pni_synapses_i3', dataset_name = 'pinky100', voxel_size = [4,4,40], sql_database_uri = 'postgresql://analysis_user:connectallthethings@swdb-em-db.crjvviai1xxh.us-west-2.rds.amazonaws.com/postgres'):
```

Comment:
might help to break to new lines after each comma to make it more readable.. otherwise looks good.